vars/folioHelm.groovy

```diff
@@ -188,6 +188,11 @@ String generateModuleValues(RancherNamespace ns, String moduleName, String modul
     moduleConfig['extraEnvVars:'] += [name: 'LEGACY_TOKEN_TENANTS', value: '']
   }
 
+  // Switch on Eureka related configuration in Helm Charts for Modules
+  if (ns.enableEureka) {
+    moduleConfig << [ eureka : [enabled: true] ]
```

Comment:
Modification is required, overwise sidecar will be inside all of the modules...
ONLY mod-* modules should have sidecar inside.